mon-pix/app/styles/pages/_challenge.scss

```diff
@@ -1,4 +1,24 @@
 .challenge {
+  /* CSS patch to delete when screen improved */
+  line-height: normal;
+
+  p {
+    margin-top: 1em;
+    margin-bottom: 1em;
+
+    p {
+      margin-top: 0;
+      margin-bottom: 0;
+    }
+  }
+
+  h2:not([class]),
+  ol:not([class]),
+  ul:not([class]) {
+    all: revert;
```

Comment:
⚠️ , le revert n'est pas pris en compte par certains navigateurs 🧓 ...  😭 

FireFox 67, mais on gère encore du FF 58 je crois côté sco pole emploi ?

Reply:
Je vois que le `:not` n'est pas valide non plus...

Reply:
C'est modifié